realm/src/main/java/io/realm/RealmObject.java

```diff
@@ -140,6 +141,10 @@ protected static Row getRow(RealmObject obj) {
         return obj.row;
     }
 
+    protected void updateColumnInfo(ColumnInfo info) {
```

Comment:
I'm still missing the use case? But at least remove `protected`, if it is `package private` users cannot accidentally  autocomplete into it and all our proxy classes are in `io.realm` anyway.